README.md

```diff
@@ -23,14 +23,13 @@
 [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-helm/depshield.svg)](https://depshield.github.io)
 
 # Table Of Contents
+* [Release notes](https://help.sonatype.com/display/NXRM3/2020+Release+Notes)
```

Comment:
Q: Is the point here to give the 2020 page or to target the release of Helm (likely 3.21 anchor but TBD?)?

Reply:
In the latter case, I suspect you could placehold:
https://help.sonatype.com/display/NXRM3/2020+Release+Notes#id-2020ReleaseNotes-RepositoryManager3.20.1

Reply:
@joedragons  why ...RepositoryManager3.20.1?  I suspect it should be ...RepositoryManager**3.21.0**. Because the next version  of NXRM should be 3.21 
Correct me if  I wrong, please.

Reply:
You're right @aornatovskyy .  I got mixed up with the upcoming point release=)
https://help.sonatype.com/display/NXRM3/2020+Release+Notes#id-2020ReleaseNotes-RepositoryManager3.21.0

Reply:
Fixed
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/107/commits/758bc38c78dcb549c4e303c1ff64dd2df45bb3b8